OnlineShopUI/online-shop-ui/src/routers/Home/Home.js

```diff
@@ -7,16 +7,19 @@ import SideMenuList from '../../components/SideMenuList/SideMenuList';
 import Banner from '../../components/Banner/Banner';
 import useMenuFilling from '../Home/Logic/MenuLogic';
 import useProductCardFilling from '../Home/Logic/ProductLogic';
-
+import Basket from '../../components/Basket/Basket';
+import { useState } from 'react';
 function Home() {
   const menu = useMenuFilling();
   const productCards = useProductCardFilling();
-
+  const [cartOpened, setCartOpened] = useState(false);
```

Comment:
It's better to start bool variables with 'is' or 'can'. For instance 'isCartOpen'. Also for use state it's better to () => false